source/Calamari/Commands/DownloadPackageCommand.cs

```diff
@@ -152,7 +153,7 @@ public override int Execute(string[] commandLineArguments)
                 Guard.NotNullOrWhiteSpace(feedUri, "No feed URI was specified. Please pass --feedUri https://url/to/nuget/feed");
             }
 
-            version = VersionFactory.TryCreateVersion(packageVersion, versionFormat);
+            version = VersionFactory.TryCreateVersion(versionFormat == VersionFormat.Semver ? packageVersion.SanitiseSemVerString() : this.packageVersion, versionFormat);
```

Comment:
Is the additional `this` required here?

Reply:
Ah good call, not sure what happened there 🤔

Reply:
Removed 👍